File: Python/dividindo_x_po_y.py
n=int(input())
cont = 1

while cont <= n:
    x=int(input())
    y=int(input())

    if  y == 0:
        print("Divisão impossivel")
    else:
        print(f"{((x*1.0) / y):.1f}")
    cont += 1

File: C/dividindo_x_po_y.cpp
#include<stdio.h>

int main(void){
	int n,x,y;
	int cont = 1;
	scanf("%d", &n);
	
	while(cont <= n){
		scanf("%d %d", &x, &y);
		if(y == 0){
			printf("Divisao impossivel\n");
		}
		else{
			printf("%.1f\n", (x*1.0)/y);
		}
		cont++;
	}
	return 0;
}
